src/main/java/org/dependencytrack/tasks/OsvDownloadTask.java

```diff
@@ -79,20 +106,97 @@
 
 public class OsvDownloadTask implements LoggableSubscriber {
 
+    public static final Path DEFAULT_OSV_MIRROR_DIR = Config.getInstance().getDataDirectorty().toPath()
+            .resolve("osv").toAbsolutePath();
+    // Max size for ecosystem zip files: 1 GiB, some ecosystem files might reach this size in a few years
+    // (Ubuntu is currently the largest one at ~380 MiB)
+    private static final long MAX_ZIP_BYTES = 1024 * 1024 * 1024;
```

Comment:
I think this is sensible. I don't envision large files becoming a big issue, but it's always good to have limits in place in case our assumptions are proven wrong.